src/app/(post-login)/submission/program/[shortName]/clinical-data/ClinicalEntityDataTable.tsx

```diff
@@ -475,10 +475,12 @@ const ClinicalEntityDataTable = ({
 			.sort(sortEntityData);
 	}
 
+	const styleThickBorderString = `3px solid ${theme.colors.grey}`;
 	const getHeaderBorder = (key) =>
 		(showCompletionStats && key === completionColumnHeaders.followUps) ||
-		(!showCompletionStats && key === 'donor_id')
-			? `3px solid ${theme.colors.grey}`
+		(!showCompletionStats && key === 'donor_id') ||
+		key === 'FO'
```

Comment:
what's `FO` in this context? may be useful to abstract this into a declaratively named constant, to facilitate reading.

Reply:
There's logic like this all over this file.